src/luascript.cpp

```diff
@@ -12887,6 +12888,23 @@ int LuaScriptInterface::luaMonsterTypeIsAttackable(lua_State* L)
 	return 1;
 }
 
+int LuaScriptInterface::luaMonsterTypeIsChallengeable(lua_State* L)
+{
+	// get: monsterType:isChallengeable() set: monsterType:isChallengeable(bool)
```

Comment:
Do we use this pattern of getter/setter being the same function name anywhere else?
It's simple and nice, but could be confusing to read sometimes.

Reply:
Yes, I think it was introduced with the revscriptsys.